packages/backend/src/scheduler/SchedulerWorker.ts

```diff
@@ -109,34 +99,23 @@ export class SchedulerWorker {
                     payload: any,
                     helpers: JobHelpers,
                 ) => {
-                    Logger.info(
-                        `Processing sendSlackNotification job "${helpers.job.id}"`,
-                        payload,
-                    );
                     await sendSlackNotification(helpers.job.id, payload);
                 },
                 sendEmailNotification: async (
                     payload: any,
                     helpers: JobHelpers,
                 ) => {
-                    Logger.info(
-                        `Processing sendEmailNotification job "${helpers.job.id}"`,
-                        payload,
```

Comment:
looks like we used to log the payload don't we want that anymore? I saw they can get pretty big so it would be messy.

Reply:
Yes, we can have the payload metadata in the start event. Note that the metadata is only available in the json logs `logs/all.json.log`. Terminal logs only render the string message.